src/components/Dashboard/FilingHistoryList/bodies/FutureEffective.vue

```diff
@@ -1,15 +1,19 @@
 <template>
-  <div v-if="filing" class="future-effective-details body-2">
+  <div class="future-effective body-2">
     <h4>{{_.subtitle}}</h4>
 
     <p>
-      The {{_.filingLabel}} date and time for {{this.getLegalName || 'this company'}}
+      The {{_.filingLabel}} date and time for {{getLegalName || 'this company'}}
       will be <strong>{{effectiveDateTime}}</strong>.
     </p>
 
-    <p v-if="filing.courtOrderNumber">Court Order Number: {{filing.courtOrderNumber}}</p>
+    <p v-if="fileNumber">
+      Court Order Number: {{filing.courtOrderNumber}}
+    </p>
 
-    <p v-if="filing.isArrangement">Pursuant to a Plan of Arrangement</p>
+    <p v-if="hasEffectOfOrder">
+      Pursuant to a Plan of Arrangement
+    </p>
```

Comment:
It looks like there's some duplication between FutureEffective and FutureEffectivePending.  What do you think about separating the elements and listing them in the parent?  Something like this:

```javascript
<FutureEffective>
  <h4>{{ subtitle }}</h4>
  <EffectiveDateTimeString />
  <CourtOrder />
  <PlanOfArrangement />
  <OneHourToProcess v-if="isFutureEffectivePending" />
  <WithDrawalNotice v-if="isFutureEffective" />
  <ContactInfo />
</FutureEffective>
```

Reply:
There's actually more duplication than you noted above, but I really don't want to break up things in components with just a few lines. How about a new ticket to tackle this (as part of future work probably)?

Reply:
Done:  bcgov/entity#15822